services/src/test/resources/describe/wdl_1_0/arrays/workflow.wdl

```diff
@@ -7,6 +7,8 @@ workflow arrays {
     Array[Int]? optional_array
     Array[Int?] array_of_optionals
     Array[Int?]? optional_array_of_optionals
+    Array[Int]+ nonempty_array
+    Array[Int]+? optional_nonempty_array
```

Comment:
What is `optional non-empty array`? Is it that if you have an array it should be non-empty, else no array at all? Similar to `Option[Array[Int]+]`?

Reply:
Yes, that's right - an array that may or may not be defined - but if it is, then it must be nonempty.

Reply:
Sounds good. Thank you for the explanation!